### DISCUSSION TITLE
Import plugin from private repository

### DISCUSSION THREAD

**Discussion by RuBAN-GT** [yarnpkg/berry#3098] [Q&A]
Hi everyone,
I have a question about yarn v2 `plugin import` cli. Is it possible to import plugin (binary) from private repository?

Thank you!

**Comment by RuBAN-GT** [comment]
Any updates?

**Reply by ramonbrugman** [reply]
Depends on what happend with it. Is it unchanged you can use it  be sure to make a copy

**Reply by RuBAN-GT** [reply]
You meant that if my repo is private I can only copy my plugin manually?

**Comment by arcanis** [comment]
We have no particular support for authentication at the moment. I guess the easiest at the moment would be to either recommand your users to copy the `raw` url from GitHub (it'll include their token, so they'll be able to pass that to `yarn plugin import`), or tell them to first download it and run `yarn plugin import` with the file path.
